src/components/film-card/film-card.jsx

```diff
@@ -14,21 +14,34 @@ const FilmCard = (props) => {
       id,
       name,
       previewImg,
-      videoSrc
+      videoSrc,
     },
-    onFilmCardMouseOver
+    isCardActive,
+    onItemInteraction,
+    onItemInteractionEnd
   } = props;
 
   return (
     <article
       className="small-movie-card catalog__movies-card"
-      onMouseOver={() => onFilmCardMouseOver(id)}
+      onMouseEnter={() => onItemInteraction(id)}
+      onMouseLeave={onItemInteractionEnd}
     >
       <Link to={`/films/${id}`} className="small-movie-card__link">
-        <VideoPlayerHOC
-          previewImg={previewImg}
-          videoSrc={videoSrc}
-        />
+        <div className="small-movie-card__image">
+          {isCardActive === true &&
+                <VideoPlayerHOC
+                  previewImg={previewImg}
+                  videoSrc={videoSrc}
+                />
+          }
+          {isCardActive === false &&
```

Comment:
```suggestion
          {!isCardActive &&
```

а вообще, если у нас есть две разметки на true/false, то логичнее использовать тернарный оператор